### ISSUE TITLE
Postman does not save response if I switch to another saved request on the left

### ISSUE THREAD

**Issue by relgames** [a85/POSTMan-Chrome-Extension#2372]
1. Postman Version: 4.7.2
2. App (Chrome app or Mac app): both
3. OS details: OSX El Capitan
4. Is the Interceptor on and enabled in the app: No
5. Did you encounter this recently, or has this bug always been there: I don't know
6. Expected behaviour: Response should not go away when switching between requests on the left tab

That's the only issue that keeps me from using Postman; it is really inconvenient to use tabs instead: I need to remember to always open a tab when I want to switch to another request.

Basically, tabs duplicate the left panel. Instead, tabs can be "sessions": when a new tab is open, all requests on the left do not have responses, but in an already open tab switching between requests should not loose responses.

**Comment by sdnts** [comment]
@relgames If I understand correctly, you'd prefer if new requests to opened up in a new tab, right?

**Comment by relgames** [comment]
@madebysid that can make it too, yes.

**Comment by a85** [comment]
There is a setting for this now in native apps.

**Comment by nmaxcom** [comment]
Exactly what says on the title happens to me as well, and it shouldn't.

On the left side, where it says History there's list of items. Basically the requests I run. But they only "remember" what I requested but not the response which is an unnecessary waste of time and resources. I'm forced to re-send my request every time I want to check my history.

**Comment by sdnts** [comment]
@nmaxcom The original request was for requests that are saved in a collection. We currently do not store responses for history requests, but I'll add this as a feature request. 
If you find yourself visiting the same URL a lot of times, you should consider making a collection and saving a response to it. I'm linking docs for [collections](https://www.getpostman.com/docs/collections) & [saved responses](https://www.getpostman.com/docs/responses)

**Comment by nmaxcom** [comment]
I'll definitely wait for that feature, as the design stands right now it doesn't make much sense without it.

Thanks.

**Comment by bhawna2109** [comment]
Tracking this here #622
